releasenotes/notes/ambient-dns-on.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: release-notes/v2
+kind: feature
+area: traffic-management
+releaseNotes:
+  - |
+    **Promoted** the `cni.ambient.dnsCapture` value to default to `true`.
+    This enables the DNS proxying for workloads in ambient mesh by default, improving security, performance, and enabling
+    a number of features.
+    This can be disabled explicitly or with `compatibilityVersion=1.24`.
+    Note: only new pods will have DNS enabled. To enable for existing pods, they must be restarted.
```

Comment:
I still think we should mention this in both `releaseNotes` _and_ `upgradeNotes`, as people might look in either spot, and might reasonably expect this kind of upgrade-specific thing to be mentioned on the `upgradeNotes` page of the release, but I'm not going to fuss too much about it.